scripts/seedData.ts

```diff
@@ -0,0 +1,207 @@
+import contentfulManagement, { Environment } from 'contentful-management';
+import fs from 'fs';
+import { join, dirname } from 'path';
+import { fileURLToPath } from 'url';
+import * as dotenv from 'dotenv';
+
+type GenreMap = {
+	[genre: string]: string;
+};
+
+type Movie = {
+	title: string;
+	posterURL: string;
+	summary: string;
+	duration: string;
+	director: string[];
+	mainActors: string[];
+	genres: string[];
+	datePublished: string;
+	rating: string;
+	ratingValue: number;
+	bestRating: number;
+	worstRating: number;
+	writers: string[];
+};
+
+type Sys = {
+	id: string;
+	linkType: string;
+	type: string;
+};
+
+type MovieGenres = Record<string, { sys: Sys }[]>;
+
+// load dotenv config
+dotenv.config();
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+
+const { CONTENTFUL_CONTENT_MANAGEMENT_API_TOKEN, CONTENTFUL_SPACE_ID, CONTENTFUL_ENVIRONMENT } =
+	process.env;
+
+const seedData = JSON.parse(fs.readFileSync(join(__dirname, 'seedData.json'), 'utf8'));
```

Comment:
This maybe could have just been:

```
import seedData from './seedData.json';
```

but if webpack/esbuild doesn't support the json loading pattern, try:

```
const seedData = require('./seedData.json');
```